databricks_cli/configure/cli.py

```diff
@@ -32,18 +32,27 @@
 
 PROMPT_HOST = 'Databricks Host (should begin with https://)'
 PROMPT_USERNAME = 'Username'
-PROMPT_PASSWORD = 'Password' #  NOQA
+PROMPT_PASSWORD = 'Password [enter stdin to read from stdin every time]' #  NOQA
 PROMPT_TOKEN = 'Token' #  NOQA
 
 
 def _configure_cli_token(profile, insecure):
     config = ProfileConfigProvider(profile).get_config() or DatabricksConfig.empty()
     host = click.prompt(PROMPT_HOST, default=config.host, type=_DbfsHost())
-    token = click.prompt(PROMPT_TOKEN, default=config.token)
+    token = click.prompt(PROMPT_TOKEN, default=config.token, hide_input=True)
     new_config = DatabricksConfig.from_token(host, token, insecure)
     update_and_persist_config(profile, new_config)
 
 
+def _configure_cli_stdin(profile, insecure):
+    config = ProfileConfigProvider(profile).get_config() or DatabricksConfig.empty()
+    host = click.prompt(PROMPT_HOST, default=config.host, type=_DbfsHost())
+    username = click.prompt(PROMPT_USERNAME, default=config.username)
+    password = 'stdin'
```

Comment:
What if my password is `stdin`? I know this password is probably not allowed in most sign up workflows in Databricks but I'd rather not special case this password..

Can we just add a flag called `read_password_from_stdin` or something?

Reply:
excellent point, let me rework this that way.
How about empty password?  Would that be an ok choice or would you rather use a flag?

Reply:
I think a flag is probably just the cleanest way.